.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+- [ ]  This bug hasn't been reported before 
+
+**Bug description**
+
+(A concise description of the bug encountered ) 
```

Comment:
Convert this into a comment by replacing it with `<!-- (A concise description of the bug encountered ) -->`